lte/gateway/python/magma/pipelined/ng_manager/session_state_manager_util.py

```diff
@@ -0,0 +1,77 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from collections import namedtuple
+
+FARRuleEntry =  namedtuple(
+                   'FARRuleEntry',
+                    ['apply_action', 'o_teid', 'gnb_ip_addr'])
+
+PDRRuleEntry = namedtuple(
```

Comment:
I'd encourage to better use `NamedTuple` from `typing` module, since it has included benefits for type hinting and checking that the `namedtuple` counterpart doesn't have.

Reply:
Comments addressed.